### ISSUE TITLE
close this issues

### ISSUE THREAD

**Issue by jun0234** [Ltu-Bike-trailer/control-software#14]
# A simple issue template

This is not a mandatory format but might be a nice to have for the first few issues.

## A task

```md
# Name of the feature we want to include

Why is this change needed and what is the change?

## Draft

If you have a draft implementation put this here, if you have any specific design requirements put that here.

## Roadmap

Add what needs to be done if it is possible to create a roadmap

- [ ] thing 1
- [x] thing 2

## Depends on

If this change requires any other issue to be closed before we can start work on this please mention them here.

## Blocking

If we are blocking any other change that we know of please mention this here.
```

## A discussion

``````md
# Name what you want to discuss

What is the issue that we are facing or what is it that you want to discuss.

## Draft

If you have a proposal for a resolution to the discussion that you want discuss
please include it here.

## Code example

If you have any code that you want to use to exemplify your point please include that here.

```language name
The code that you want to make pretty.
```

## Relevant issues

Please link to any other relevant issues here
``````
